### DISCUSSION TITLE
Is there a way to accelerate the A100 on colab?

### DISCUSSION THREAD

**Discussion by grexzen** [TheLastBen/fast-stable-diffusion#1058] [Q&A] [answered]
Getting 15 it/s now.

This posts says the A100 can get up to 50it/s. 

https://www.reddit.com/r/MachineLearning/comments/z9q0pq/p_probably_the_fastest_open_source_stable/

Is that possible on Colab? 

Thanks! 

![image](https://user-images.githubusercontent.com/30579087/208217508-b1756b99-0790-45a1-a32b-1e615264ca7b.png)

**Comment by TheLastBen** [comment, marked as answer]
tried the oneflow, it's not up to date, I think it's broken at the moment
